Header header
DynamixelState[] dynamixel_state
